perfkitbenchmarker/providers/ibmcloud/flags.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing flags applicable across benchmark run on IBM Cloud."""
+
+from absl import flags
+
+flags.DEFINE_string('ibmcloud_region', 'us-south',
+                    'Name of IBM Cloud Location or region.')
+
+flags.DEFINE_string('ibmcloud_zones', 'us-south-1',
```

Comment:
Can you use the regular FLAGS.zones ?  It would be nice to not have to not have to specify zones in a special way for a provider.

Reply:
yes I agree, updated.